src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/SpecialFileProviders/AppDesignerFolderSpecialFileProvider.cs

```diff
@@ -1,82 +1,55 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System;
 using System.ComponentModel.Composition;
 using System.IO;
 using System.Linq;
-using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.VisualStudio.ProjectSystem.SpecialFileProviders
 {
     /// <summary>
-    ///     Provides a <see cref="ISpecialFileProvider"/> that handles the AppDesigner, 
+    ///     Provides a <see cref="ISpecialFileProvider"/> that handles the AppDesigner folder;
     ///     called "Properties" in C# and "My Project" in Visual Basic.
     /// </summary>
     [ExportSpecialFileProvider(SpecialFiles.AppDesigner)]
     [Export(typeof(AppDesignerFolderSpecialFileProvider))]
```

Comment:
Nit: Looks like this export could be removed

Reply:
Yep, you are right.